txpool/txpool.go

```diff
@@ -670,7 +670,6 @@ func (p *TxPool) validateTx(tx *types.Transaction) error {
 		}
 	}
 
-	// Check if the given tx is not underpriced
 	if tx.GetGasPrice(baseFee).Cmp(new(big.Int).SetUint64(p.priceLimit)) < 0 {
```

Comment:
Revert comment, IMO it is useful to have it there:

```suggestion
	// Make sure that the transaction is not underpriced
```